server/etcdserver/server.go

```diff
@@ -839,7 +839,7 @@ func (s *EtcdServer) run() {
 	for {
 		select {
 		case ap := <-s.r.apply():
-			f := func(context.Context) { s.applyAll(&ep, &ap) }
+			f := schedule.NewJob("server applyAll", func(context.Context) { s.applyAll(&ep, &ap) })
```

Comment:
nit: I would avoid using 'space' in the identifiers. 

I would prefer any of:  "server.applyAll" or "server:applyAll" or "server_applyAll" as the convention for naming.   If we ever used this IDs for monitoring metric's field, it would also make things cleaner.

Reply:
I have changed it.